README.md

```diff
@@ -1 +1,34 @@
-# rse_cdt_project2022
\ No newline at end of file
+# CDT group project 
+
+## Finding the area of a circle given the radius, without using pi
+
+### List of contributors
+ 
+
+
+| Issue No | Project                                                                                                                                | Assignee                                                  |
+| ----- | -------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------- |
+| 1     | Writing the README      | Jo                |
+| 2     | Adding a license        |  Meg |
```

Comment:
Missing initial issue (I believe it was #2) - Kasia worked on creating .gitignore file